src/Firebase/firebase-config.js

```diff
@@ -0,0 +1,45 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from 'firebase/app';
+import { getAuth, GoogleAuthProvider, signInWithPopup } from 'firebase/auth';
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: "AIzaSyCV0H3GLYarzZEhiZ-Dwoyda5Fu5d4x0E0",
```

Comment:
you may not want have apikey in your frontend. you also cannot use .env because it is still going to be embeded into the build file and accessible from users. 
apparently the way to go is have create a new api endpoint on the serverside code and use .env .
reference:  https://stackoverflow.com/questions/48699820/how-do-i-hide-api-key-in-create-react-app

Reply:
the firebaseConfig can be included on the client side. I just need to modify some firebase server settings to secure the api.

Reply:
sounds good!!! thanks for working on it